engine/class_modules/sc_death_knight.cpp

```diff
@@ -4846,11 +4858,7 @@ struct frostscythe_t : public death_knight_melee_attack_t
     parse_options( options_str );
 
     weapon = &( player -> main_hand_weapon );
-    aoe = -1;
-
-    weapon_req = WEAPON_1H;
-
-    crit_bonus_multiplier *= 1.0 + p -> spec.death_knight -> effectN( 5 ).percent();
```

Comment:
Could add a comment mentionning that the crit multiplier is handled in apply_affecting_auras()

Reply:
Great idea, added in